hbase/pom.xml

```diff
@@ -75,11 +75,23 @@
       <groupId>org.apache.hbase</groupId>
       <artifactId>hbase-client</artifactId>
       <version>${hbase.hbase.version}</version>
+      <exclusions>
```

Comment:
This is not a test dependency, so the exclude should be unnecessary. Normal dependencies do not contain test libraries.